.github/shared.py

```diff
@@ -0,0 +1,27 @@
+from typing import Iterable
+
+import yaml
+
+WORKFLOW_PATH = '.github/workflows/continuous-integration.yml'
+REMAINDER = 'REMAINDER'
```

Comment:
It would be great if you can add some description about `REMAINDER` and why/how it's used.

Reply:
I've added comment elsewhere. Please review again from the workflow file.